<frozen importlib._bootstrap>:219: RuntimeWarning: scipy._lib.messagestream.MessageStream size changed, may indicate binary incompatibility. Expected 56 from C header, got 64 from PyObject
Report file will be located at:  /work/mech-ai/bella/MolTransformer/test_output/save_noneddp_qm9/multiF_HF_log.log
imported
self.mode multiF_HF
done intialized
load the cpu model!! 
Train paths:  ['/work/mech-ai/bella/MolTransformer/model/utils/../data/qm9/train/train_qm9.csv']
Test paths:  ['/work/mech-ai/bella/MolTransformer/model/utils/../data/qm9/test/test_qm9.csv']
in file:  /work/mech-ai/bella/MolTransformer/model/utils/../data/qm9/train/train_qm9.csv
number of train 107106
in file:  /work/mech-ai/bella/MolTransformer/model/utils/../data/qm9/test/test_qm9.csv
number of test 26776
not ss
start compute descriptors
start dataset_building
end char2ind
end dataset_test
---------build model operator-----------
----------------------mae:  0.1325127
shape of mae (640,)
type of mae <class 'numpy.float32'>
